docs/php.md

```diff
@@ -53,8 +53,9 @@ jobs:
       PHPCS_ARGS: '--report=summary'
 ```
 
-**Note**: Coding standards analysis can only be performed with a specific PHP version and not in a PHP matrix, as it should always be tested with the highest PHP version in use. To check compatibility with multiple PHP versions, use the [Lint PHP](#lint-php) workflow.
-
+**Note**: Coding standards analysis can only be performed with a specific PHP version and not in a
```

Comment:
> Coding standards analysis can only be performed with a specific PHP version and not in a PHP matrix

But it can be in a matrix through the calling workflow, right? 😅

Reply:
How about now?